app/models/ip.rb

```diff
@@ -1,7 +1,7 @@
 class Ip < ApplicationRecord
   belongs_to :location
 
-  validates :address, presence: true, uniqueness: true
+  validates :address, presence: true, uniqueness: { case_sensitive: true }
```

Comment:
Is this needed as part of the upgrade? Also, the default is `true` anyway.

Reply:
```
.DEPRECATION WARNING: Uniqueness validator will no longer enforce case sensitive comparison in
 Rails 6.1. To continue case sensitive comparison on the :address attribute in Ip model, pass 
`case_sensitive: true` option explicitly to the uniqueness validator. (called from block (4 le
vels) in <top (required)> at /usr/src/app/spec/models/ip_spec.rb:66)
```